package.json

```
@@ -27,7 +27,9 @@
     "postinstall": "git submodule update --init --recursive",
     "test:anvil": "npx tsx ./tests/anvil.ts",
     "test": "jest",
-    "clean": "rm -rf dist"
+    "clean": "rm -rf dist",
+    "test:script:standard": "npx tsx tests/script-test.ts",
+    "test:script:consensus": "npx tsx tests/script-test-consensus.ts"
```

```suggestion
    "test:script:standard": "bun tests/script-test.ts",
    "test:script:consensus": "bun tests/script-test-consensus.ts"
```

This seems to be more concise especially since we are mostly migrated to bun these days.

---

You need to do `bun x tsx` as just `bun` will hang. `bun x ` pulls `tsx` the same way `npx` does.

---

https://github.com/ubiquity/rpc-handler/issues/71

I made a new issue for it specifically as it's more work than just your suggestion and is a bit off spec.